src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/NoWhitespaceBeforeCheckTest.java

```diff
@@ -126,4 +126,14 @@ public void testEmptyForLoop() throws Exception {
         verify(checkConfig, getPath("InputNoWhitespaceBeforeEmptyForLoop.java"), expected);
     }
 
+    @Test
+    public void testLabel() throws Exception {
+        final DefaultConfiguration checkConfig = createModuleConfig(NoWhitespaceBeforeCheck.class);
+        checkConfig.addAttribute("tokens", "LABELED_STAT");
+        final String[] expected = {
+            "6:16: " + getCheckMessage(MSG_KEY, ":"),
+        };
+        verify(checkConfig, getPath("InputNoWhitespaceBeforeLabel.java"), expected);
```

Comment:
again, you can just extend UT for default config, no need for separate test case

Reply:
Done.